widget.py

```diff
@@ -0,0 +1,309 @@
+"""
+Module for centering switches and handling mouse events.
+"""
+import itertools
+from matplotlib.widgets import Button, SpanSelector, Slider, RadioButtons
+import crystlib
+
+
+class Bttn_bins(Button):
+    """
+    Klasa reprezentująca guzik do zmiany ilości
```

Comment:
polish comments

Reply:
Addressed in: e73a1996e75cfceb523bf223bbad6228e6e19c81 e44d32b121a71430393d2698681a687cc057acb3